### ISSUE TITLE
Networking

### ISSUE THREAD

**Issue by d0zzer2** [kicker22004/ECO_LINUX_SERVER_MANAGER#6]
I'm writing this here and not on the ECO forums, looked like you were more active here.

All is set up and well. I can connect to the server and web server via LAN and I can connect to the web server from external means but I can't connect to my server with the external ip address. Default ports are forwarded in the router. I'm not sure what else to check, I'm also new to Linux so I have no idea if it's related to that either. Thanks for any help!

**Comment by kicker22004** [comment]
So your saying that the webUi is live but you can't see it in your list? And this server is running from your house?

**Comment by d0zzer2** [comment]
Correct. Both client computer (Windows 10) and server (latest Ubuntu) are behind the same router. I can connect over LAN to the game and the webpage but when I add server in game and use the external IP it shows offline.

**Comment by d0zzer2** [comment]
I can turn WIFI off on my phone and connect to the web page via external IP also.

**Comment by kicker22004** [comment]
yeah you shouldn't need to add it on the client, it should list under LAN on the client. Ok the Phone is verifying that it's ports are working. Can you check mark the Show incompatible versions on your client with the server running please.

**Comment by d0zzer2** [comment]
It does list under LAN right at the top, I guess i was trying to verify someone could connect to it with the external ip before I gave them the address. Is that not possible?

**Comment by kicker22004** [comment]
well If you PM me on discord I cant attempt to connect for you.
